sources/packages/api/src/services/education-program-offering/custom-validators/program-allows-offering-delivery.ts

```diff
@@ -44,18 +44,24 @@ class ProgramAllowsOfferingDeliveryConstraint
   }
 
   defaultMessage(args: ValidationArguments) {
-    return `${args.property} has an offering delivery that is not allowed by its program.`;
+    const [propertyDisplayName] = args.constraints;
+    return `${
+      propertyDisplayName ?? args.property
```

Comment:
what value will `args.property` have? for my info

Reply:
The decorated property name. In the below example would be the string "nameOfTheProperty".
```ts
@SomeValidator()
nameOfTheProperty: string;
```

Reply:
okay got it